EIPS/eip-4844.md

```diff
@@ -211,7 +215,7 @@ the `TransactionNetworkPayload` version of the transaction also includes `blobs`
 The execution layer verifies the wrapper validity against the inner `TransactionPayload` after signature verification as:
 
 * All hashes in `blob_versioned_hashes` must start with the byte `BLOB_COMMITMENT_VERSION_KZG`
-* There may be at most `MAX_BLOBS_PER_TX` blob commitments in any single transaction.
+* There may be at most `MAX_BLOBS_PER_BLOCK` blob commitments in any single transaction.
```

Comment:
```suggestion
* There may be at most `MAX_BLOBS_PER_BLOCK` blob commitments in any single block.
```

Reply:
The point was to clarify that the per-tx blob number is still limited effectively. Could be reworded though to make that more clear.

Reply:
I see this is given on the next line -- I would suggest just deleting this line for now.

Reply:
done